backend/scripts/get_machines_per_location.py

```diff
@@ -0,0 +1,83 @@
+"""
+Country based download of HTML from PennyCollector.com
+This script does:
+1. Scraping the HTML of a location and extracting machine name/subtitle and link.
+2. Searches the location name on a map and saves the geographic coordinates
+3. Saves data to .json
+"""
+import argparse
+import json
+import os
+
+from pennyme.locations import COUNTRY_TO_CODE, parse_location_name
+from pennyme.pennycollector import (
+    LOCATION_PREFIX,
+    get_machine_list_from_locations,
+    get_raw_locations_from_location_website,
+)
+from pennyme.webconfig import get_website
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-l", "--location", type=str, help="location name to be parsed "
+)
+parser.add_argument("-a", "--api_key", type=str, help="Google Maps API key")
+parser.add_argument(
+    "-o", "--output_folder", type=str, help="Output folder path"
+)
+parser.add_argument(
+    "-i",
+    "--start_id",
+    type=int,
+    help="Machine ID from which enumeration is started",
```

Comment:
Why is this an argument? Can't we read the current json and use the maximum ID +1?

Reply:
Okay, I guess this is just a script for a single machine, so it's not necessary to automate the ID assignment

Reply:
This is legacy code